### DISCUSSION TITLE
now playing not recognized because of extra info

### DISCUSSION THREAD

**Discussion by TinkeringB** [rern/rAudio#1087] [Feature Requests]
Dutch web-radio station "IndieXL"  (https://www.mp3streams.nl/zender/indiexl/stream/124-mp3-192) provides the release year in the now playing information of a track. As a result, no coverart or lyrics are available. 
![image](https://user-images.githubusercontent.com/9662310/199936473-50503978-b5d6-4c3b-a7a5-2425c85a06af.png)

would it be possible and feasible to implement a way to make the recognition process to ignore certain patterns, like the release-year in the now playing info?

**Comment by R101G** [comment]
@bstrijker 
That might be a problem for Prince fans.

**Reply by TinkeringB** [reply]
I've been told everything is possible in software, but challenges will exist always.

Would it be possible to personalize something like a filter pattern using the User's Configurations somehow?
![image](https://user-images.githubusercontent.com/9662310/199975957-c814cf09-1d16-4722-8c2e-47a0fe73eb22.png)

or - is is simply too difficult/ not worth the time? It's just I am enjoying the now-playing coverart etc. so much, I thought I'd give it a try.

**Comment by R101G** [comment]
I think would be easier to ask the radio station to put the track-title in the track-title field, and to put the year in the year field. That is what most stations do. Unless it's a way to avoid paying royalties.

**Reply by TinkeringB** [reply]
hmm.. more research then.

About the insinuation that this station would do this deliberately to avoid paying royalties - can you elaborate?

**Reply by skycamel** [reply]
I actually think it's the other way around, namely to protect royalties.
Many stations add/modify/hide titles or performers and also don't update the information at the beginning of the song to make it more difficult to rip songs into separated music files.

**Comment by R101G** [comment]
Metadata is often used to identify the rights holder, but I am sure there are many reasons why a perfectly respectable radio station would want to change the name of the track they are playing. I am sure they will tell you if you ask them. The BBC don't send any data at all, because they want everyone to use their Sounds app.

**Comment by skycamel** [comment]
As far as the lyrics is concerned, you can update the metadata manually. After you click on the song title, just delete whatever doesn't supposed to be there.

**Comment by TinkeringB** [comment]
Thank you @rern :-) - Really enjoying the update as we speak!! 

![afbeelding](https://user-images.githubusercontent.com/9662310/202799678-96e0ef99-e1bd-4ed1-afdf-5b6952b8ed69.png)
